tests/test_comments.py

```diff
@@ -18,8 +18,14 @@ def test_add_comment(self):
         text = 'Unique comment: {0}'.format(self.r.modhash)
         submission = next(self.subreddit.get_new())
         comment = submission.add_comment(text)
-        self.assertEqual(comment.submission, submission)
+        self.assertEqual(comment.get_tree(), submission)
```

Comment:
Can you pull these out into their own tests? That way we can be sure the original `add_comment` test still works as is with all of the changes (`test_add_comment` shouldn't require any additional requests).